website/views/options/settings.jade

```diff
@@ -74,17 +74,15 @@ script(type='text/ng-template', id='partials/options.settings.settings.html')
             button.btn.btn-default(ng-if='!user.preferences.disableClasses && user.flags.classSelected', ng-click='showClassesTour()', popover-trigger='mouseenter', popover-placement='right', popover=env.t('classTourPop'))= env.t('showClass')
 
             div.alert.alert-warning(style='padding:2px;margin-top:7px')
-              h5.hint(popover=env.t('clockInfo'), popover-trigger='mouseenter')=env.t('customDayStart')
+              h5=env.t('customDayStart')
+              h5=env.t('customDayStartInfo1')
+                a(ng-show='!showCustomDayStartInfo' ng-click='showCustomDayStartInfo = !showCustomDayStartInfo')=env.t('customDayStartInfo2')
+                a(ng-show='showCustomDayStartInfo' ng-click='showCustomDayStartInfo = !showCustomDayStartInfo')=env.t('customDayStartInfo3')
```

Comment:
This works great, but I think we can make a single `a` tag do all the work instead of two. How about:

```
h5=env.t('customDayStartInfo1')
  a(ng-click='showCustomDayStartInfo = !showCustomDayStartInfo') {{!showCustomDayStartInfo ? env.t('customDayStartInfo2') : env.t('customDayStartInfo3')}}
```

Not sure how familiar you are with ternary operators, but basically that'll check the truthiness of `showCustomerDayStartInfo`, if it's `false`, it'll use the "Show More" text. If it's `true`, it'll use the "Show Less" text. 

Does that make sense?